src/dashboard/Apps/AppsIndex.react.js

```diff
@@ -156,7 +158,7 @@ export default class AppsIndex extends React.Component {
         <ul className={styles.apps}>
           {apps.map(app =>
             app.name.toLowerCase().indexOf(search) > -1 ?
-              <AppCard key={app.slug} app={app} /> :
+              <AppCard key={app.slug} app={app} icon={this.props.iconsFolder && app.icon ? this.props.iconsFolder + '/' + app.icon : null}/> :
```

Comment:
I think we probably want to not have the client know about the `iconsFolder`. I'd suggest you use a consistent path like `/appicons/${appId}` and then in `app.js` add a `appicons` route and serve up the image based on the `appId` in the parameter. Then you don't need to worry about the icons folder being publicly accessible.